docs/GettingStarted.md

```diff
@@ -352,3 +376,99 @@ issions are:
+
+If you are running in the dev container, you are in an isolated environment with no access to this filesystem.
+On most systems, you can add this as an explicit mount by adding the following to the [`devcontainer.json`](https://github.com/microsoft/cheriot-rtos/blob/main/.devcontainer/devcontainer.json) file:
+
+```json
+  "mounts": [
+    "source={/your/SONATA/mount/point},target=/mnt/SONATA,type=bind"
+  ]
+```
+
+Replacing `{/your/SONATA/mount/point}` with the mount point of your `SONATA` filesystem.
+Your editor should prompt to rebuild the container at this point.
+
+If you invoked the dev container directly, rather than via an editor such as VS Code, then you will need to instead add the following to your `docker run` command:
+
+```
+--mount source={/your/SONATA/mount/point},target=/mnt/SONATA,type=bind
+```
+
+Again, replacing `{/your/SONATA/mount/point}` with the real mount point.
+
+If you are on Windows, this will not work because Docker on Windows runs containers in a WSL2 VM and needs to have host filesystems mounted explicitly (Docker on macOS does this automatically).
+In theory, the following steps should work (run in the WSL2 terminal in the Docker WSL2 VM):
+
+```
+sudo mkdir /mnt/d
+sudo mount -t drvfs D: /mnt/d/
+```
+
+Assuming that `SONATA` is mounted as `D:` on Windows.
+The above commands should then work.
+Unfortunately, they do not.
+The cause of this is not currently known but will hopefully be fixed in a newer version of the Sonata RPi2040 firmware.
+
+### Seeing UART output
+
+If you ran the above `xmake run` commands, you will have noted that there is no output from the test suite.
+When you use a simulator, the UART output goes directly to the simulator's standard output.
+In contrast, when you run on an FPGA the output will go to a USB serial device.
+
+You will need to connect a terminal emulator to the corresponding serial port.
+The Sonata system exposes *four* serial devices, typically numbered 0-3.
+The UART is device 2.
+On Linux, this is usually `/dev/ttyUSB2` (if no other USB serial devices are connected).
+On macOS, the device name includes the serial number of the device and so will be something like `/dev/tty.usbserial-LN28292` (the last digit is the serial port number).
+On Windows, it will be a COM port, but unfortunately Windows is not always consistent about the order in which the ports appear.
+
+If you install a serial terminal (for example, `minicom` on *NIX or PuTTY on Windows), you can point it at the relevant serial port.
+
+The serial setup is the same as for the Arty A7 [above](#installing-and-running-the-firmware), aside from the different device name.
+
+Once a serial terminal is connected, running `xmake run` should show the output from the test suite.
+
+If you are unsure which of Sonata's serial ports is the correct one, you can connect your serial console to all four and see which produces output from the test suite.
```

Comment:
I don't think it's a good idea to connect to all of them at the same time. I think one might be the debugger and actually halts CPU or something.
```suggestion
If you are unsure which of Sonata's serial ports is the correct one, you can connect your serial console to each of them in turn and see which produces output from the test suite.
```

Reply:
> I don't think it's a good idea to connect to all of them at the same time. I think one might be the debugger and actually halts CPU or something.

I connected to all four on my Mac to figure out the right one and it was fine.